worlds/dark_souls_3/docs/locations_en.md

```diff
@@ -0,0 +1,2269 @@
+    in the furthest pantry room.
+
+* **Parapets:** The area with balconies and Overgrown Lothric Knights in
+  vanilla, accessible by taking the pillar bridge from the church, following
+  that path to the end, and dropping down to the right.
+
+* **Ruins:** The area around the Earthen Peak Ruins bonfire, up to the swamp.
+
+* **Swamp:** The area in and above the poisonous water, up to the point the
+  branches deposit you back on the ruins.
+
+  * "Left": Left as you enter from the ruins, towards the cliff edge.
+
+  * "Right": Right as you enter from the ruins, towards higher ground.
+
+  * "Upper": The path up and over the swamp towards the Within Earthen Peak
+    Ruins bonfire.
+
+### Ringed City
+
+The "mid boss", "end boss", and "hidden boss" are the bosses who take the place
+of Halflight, Gael, and Midir, respectively.
+
+* **Wall:** The large wall in which you spawn when you first enter the area,
+  with the Mausoleum Lookout bonfire.
+
+  * "Top": The open-air top of the wall, where you first spawn in.
+
+  * "Upper": The upper area of the wall, with the Ringed Inner Wall bonfire.
+
+  * "Tower": The tiered tower leading down from the upper area to the stairs.
+
+  * "Lower": The lower rooms of the wall, accessible from the lower cliff, with
+    an elevator back to upper.
+
+  * "Hidden": The hidden floor accessible from the elevator from lower to upper,
+    from which you can reach Midir in vanilla.
+
+* **Streets:** The streets and skyways of the city proper. "Left" and "right"
+  are relative to the main staircase as you head down towards the swamp, "near"
+  and "far" are relative to Shira's chamber at the top of the stairs.
+
+  * "Garden": The flower-filled back alley accessible from the left side of the
+    nearest bridge over the stairs.
+
+  * "High": The higher areas in the far left where you can find the Locust
+    Preacher, accessible from a long ladder in the swamp.
+
+  * "Monument": The area around the purging monument, which can only be accessed
+    by solving the "Show Your Humanity" puzzle.
+
+* **Swamp:** The wet area past the city streets. "Left and "right" are relative
+  to heading out from the Ringed City Streets bonfire, and "near" and "far" are
+  relative to that bonfire as well.
+
+* **Lower cliff:** The cliffside path leading from the swamp into the shared
+  grave, where Midir breathes fire.
+
+* **Grave:** The cylindrical chamber with spiral stairs around the edges,
+  connecting the two cliffs, containing the Shared Grave bonfire.
+
+* **Upper cliff:** The cliffside path leading out of the grave to the lower
+  wall.
+
+* **Church path:** The sunlit path from the lower cliff up to the Church of
+  Filianore where you fight Halflight in vanilla.
+
+* **Ashes:** The final area, where you fight Gael in vanilla.
+
+## Detailed Location Descriptions
+
+These location decsriptions were originally written by [Matt Gruen] for [the
+offline _Dark Souls III_ randomizer].
```

Comment:
I could move it into yet another different page, but then it's another pager players have to navigate to. I think it makes more sense to just have _all_ the density here. Since this is at the bottom of the page anyway, I think the actual cost of including it is pretty low despite how dense it is.

Reply:
I agree that it definitely helps with it being at the bottom. Another concern that I had was loading times for having such a large file, but after taking a look at the transfer size, the file ends up being surprisingly small, so I don't have any issues with it being here. I was more wondering if there was this table as part of the standalone.